plugins/wallet_plugin/wallet_manager.cpp

```diff
@@ -40,16 +40,17 @@ std::string wallet_manager::create(const std::string& name) {
    if (fc::exists(wallet_filename)) {
       EOS_THROW(chain::wallet_exist_exception, "Wallet with name: '${n}' already exists at ${path}", ("n", name)("path",fc::path(wallet_filename)));
    }
-
+   
    wallet_data d;
    auto wallet = make_unique<wallet_api>(d);
    wallet->set_password(password);
    wallet->set_wallet_filename(wallet_filename.string());
    wallet->unlock(password);
-   wallet->save_wallet_file();
+   static const std::string eosio_wif_key("5KQwrPbwdL6PhXujxW37FSSQZ1JiwsST4cqQzDeyXtP79zkvFD3");
```

Comment:
What do you think of making this a defaulted argument to the wallet plug so a user can specify a different key without re-compiling?

Reply:
I like this idea

Reply:
Added that